spec/codebase/codebase_spec.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Codebase', codebase: true do
+  it 'does not offend eslint' do
+    expect(`yarn run eslint`).to include 'Done'
+  end
```

Comment:
```suggestion
  it 'does NOT offend Rubocop' do
    expect(`rubocop --parallel --format simple`).to include 'no offenses detected'
  end
```

ESLint is for linting JavaScript (EcmaScript), as this project is an API one, we might only need Rubocop.


Optionnaly you can also add Brakeman (for security checks) and Zeitwerk (just the Gem loader):

```

  it 'satisfies Brakeman' do
    expect(`brakeman -w1`).not_to include '+SECURITY WARNINGS+'
  end

  it 'does NOT break zeitwerk loading' do
    expect(`bundle exec rake zeitwerk:check`).to include 'All is good!'
  end
```